projects/migrations/0013_populate_is_billable.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-06-08 10:45
+
+"""
+    For more details on this migration, see:
+
+    https://github.com/18F/projects/issues/37
+"""
+
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+BILLABLE = 0
+NON_BILLABLE = 1
+
+def convert_billable_to_boolean(apps, schema_editor):
+    Project = apps.get_model("projects", "Project")
+    for project in Project.objects.all():
+        if project.billable == BILLABLE:
+            project.is_billable = True
+        elif project.billable == NON_BILLABLE:
+            project.is_billable = False
+        else:
+            raise Exception('%s billable is %s' % (project, project.billable))
+        project.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0012_add_is_billable'),
+    ]
+
+    operations = [
+        migrations.RunPython(convert_billable_to_boolean,
+                             reverse_code=migrations.RunPython.noop)
```

Comment:
I had to make this migration reversible in order for the unit tests to work.

This raises a concern: because of the way the `MigrationTestCase` works, during setup it actually takes a fully-migrated DB and migrates its schema *backwards*. I think this means that if we ever add an irreversible migration, it'll break the tests, which is unfortunate. I guess we can get around this by either:

* Never making any irreversible migrations (not sure if this is even possible)
* Getting rid of the relevant migration test once it's clearly no longer needed
* Modifying the `MigrationTestCase` to actually wipe the db and migrate only to the needed migration, rather than starting with a fully migrating db and migrating backwards.